scaffolding-ruby/lib/scaffolding.sh

```diff
@@ -196,18 +196,24 @@ scaffolding_bundle_install() {
 
   build_line "Installing dependencies using $(_bundle --version)"
   start_sec="$SECONDS"
-  _bundle_install \
-    "$CACHE_PATH/vendor/bundle" \
-    --retry 5
+
+  {
```

Comment:
You may be to avoid the curl braces around the single `_bundle_install()` call, but it certainly doesn't hurt.